bc_events/session.py

```diff
@@ -110,6 +110,32 @@ def publish(self, action=None, entity=None, data=None, category=None):
         topic = self.client.get_topic(category, entity, action)
         self._publish(topic, data)
 
+    def publish_all(self, events):
```

Comment:
Having `self.events`, I do not see a need for a separate `events` parameter here. The current usage is mixed, and I think it would be best to stick with one source of truth for the events to be published.